packages/util/src/crypt.ts

```diff
@@ -49,12 +49,12 @@ const stringToByteArray = function(str) {
 /**
  * Turns an array of numbers into the string given by the concatenation of the
  * characters to which the numbers correspond.
- * @param {Array<number>} bytes Array of numbers representing characters.
+ * @param {number[]} bytes Array of numbers representing characters.
```

Comment:
all the param/return types should be moved from the comment to the definition directly.

Reply:
Done, fixed similar in rest of file.